Extensions/InternetConnectivity.json

```diff
@@ -0,0 +1,32 @@
+{
+  "author": "The Gem Dev",
+  "description": "",
+  "extensionNamespace": "",
+  "fullName": "Internet Connectivity",
+  "name": "InternetConnectivity",
+  "shortDescription": "an extension which checks if the device is connected to the internet. ",
+  "tags": "javascript, desktop, mobile, internet, connection",
+  "version": "0.0.1",
+  "eventsFunctions": [
+    {
+      "description": "checks if the device running the game is connected to the internet.",
```

Comment:
Capital letter:
`Checks if the device running the game is connected to the internet.`